### PR TITLE
Extend documentation for TSError

### PR DESCRIPTION
**Pull request by theHamsta**
I would love to add also a documentation here how to deactivate the error highlighting here. But I don't know how.

### CODE REVIEW

File: doc/nvim-treesitter.txt [hunk 1]
```diff
@@ -170,7 +170,9 @@ Note: This is highly experimental, and folding can break on some types of
 ==============================================================================
 HIGHLIGHTS                                        *nvim-treesitter-highlights*
 
-TSError
+`TSError`
+							          *hl-TSError*
+For syntax/parser errors.
```
Comment (kyazdani42) [inline doc/nvim-treesitter.txt:175]: is this highlight group disabled by default ? If so, you could write about that here.

Comment (theHamsta) [reply doc/nvim-treesitter.txt:175]: It's enabled. I would prefer disabled.

Comment (vigoux) [reply doc/nvim-treesitter.txt:175]: Then we can easily disable by default, can you change this in this PR ?

Comment (theHamsta) [reply doc/nvim-treesitter.txt:175]: You can change it to default=Off if you like.

**Pull request merged**